src/components/container/AIAssistant/AIAssistant.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon';
+import { faFlask } from '@fortawesome/free-solid-svg-icons/faFlask';
+import { StreamEvent } from '@langchain/core/tracers/log_stream';
+import { AIMessageChunk } from '@langchain/core/messages';
+import { StructuredTool } from '@langchain/core/tools';
+import MyDataHelps from '@careevolution/mydatahelps-js';
+
+import { MyDataHelpsAssistant } from '../../../helpers/assistant/assistant';
+import Chat from '../../presentational/Chat';
+
+import '@fortawesome/fontawesome-svg-core/styles.css';
+
+export interface AIAssistantProps {
+    innerRef?: React.Ref<HTMLDivElement>;
+    previewState?: "default";
+    debug: boolean;
+    additionalInstructions?: string;
+    tools?: StructuredTool[];
+    appendTools?: boolean;
+    baseUrl?: string;
```

Comment:
Can we just infer this from the base url of MyDataHelps.js?  Is there a reason to override it?